rollout.go

```diff
@@ -110,6 +112,27 @@ func (r *client) swapData(data []byte) error {
 	return nil
 }
 
+// RawPercentage returns the raw percentage from the rollout section
+func (r *client) RawPercentage(feature string) (float64, error) {
+	feature = "feature:" + feature
+	r.RLock()
+	value, ok := r.currentData[feature]
+	r.RUnlock()
+
+	if !ok {
+		return 0.0, fmt.Errorf("feature not found: %s", feature)
+	}
+	splitResult := strings.Split(value, "|")
+	if len(splitResult) != 3 {
+		return 0.0, fmt.Errorf("invalid value for %s: %s", feature, value)
```

Comment:
I appreciate the delta from the original code in FeatureActive. FeatureActive is probably often used directly in convenient `if rollout.FeatureActive(...)` checks, and so returning a second error argument is awkward, but I've never been happy to have logging taking place in my application that is not controlled by my chosen primary logging system's settings. (I didn't realize there were logrus logs in here!)

I can't advocate changing the other function because it would probably break a lot of client code, though. :thinking: